source/index.rst

```diff
@@ -38,6 +29,12 @@ Welcome to the FIRST Robotics Competition Documentation! This documentation is v
    docs/software/basic-programming/index
    docs/software/advanced-programming/index
 
+.. toctree::
+   :maxdepth: 1
+   :caption: API Docs
+   WPILib Java API Docs <https://first.wpi.edu/FRC/roborio/release/docs/java/index.html>
```

Comment:
Need a blank line here in between the :caption: option and the text

Reply:
Whoops. Will fix after the debate